doc/src/manual/performance-tips.md

```diff
@@ -331,7 +331,37 @@ For reasons of length the results are not shown here, but you may wish to try th
 the type is fully-specified in the first case, the compiler doesn't need to generate any code
 to resolve the type at run-time. This results in shorter and faster code.
 
-### Avoid fields with abstract containers
+One should also avoid fields with unparameterized parametric types:
+
+```jldoctest myambig3
+julia> mutable struct MyParametricType{T}
+           a::T
+       end
+       
+julia> mutable struct MySecondType{T <: MyParametricType}
+           mpt::T
+       end
+       
+julia> mutable struct MyThirdType{T}
+           mpt::MyParametricType{T}
+       end
+ ```
+`MySecondType` and `MyThirdType` are both better choices than 
+
+```jldoctest myambig3
+julia> mutable struct MyAlsoAmbiguousType
+           mpt::MyParametricType
+       end
+```
+Even though `MyParametricType` is not an abstract type, when it is not parameterized, it is not a concrete type either:
```

Comment:
> `when it is not parameterized`

I read four times until I realized this belongs to `it is not a concrete type either` 😂  Perhaps you can adjust it a bit for nonnative English speakers.

Reply:
I guess this is because I'm not native either. I changed this to something clearer.